spec/rails_helper.rb

```diff
@@ -71,3 +71,10 @@
     with.library :rails
   end
 end
+
+VCR.configure do |config|
+  config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
+  config.hook_into :webmock
+  config.filter_sensitive_data('<client_id>') { ENV.fetch('client_id') }
+  config.configure_rspec_metadata!
```

Comment:
```
config.default_cassette_options = {
    record: :new_episodes,
    re_record_interval: 7.days
  }
```

Would be a good addition eventually later. Although this configuration works fine for now, so we can potentially add this in later PR.

Reply:
this has been added in PR #24